### PR TITLE
  Mail text adapted to 2 days in advance

### PR DESCRIPTION
**Pull request by coogor**
Mail text adapted to 2 days in advance

### CODE REVIEW

**Comment by cboltz** [comment]
Please fix the merge conflict.

**Comment by coogor** [comment]
The button to mark it as resolved is greyed out.
Sorry, not too familiar with github

File: data/board-meeting.mail [hunk 1]
```diff
@@ -1,11 +1,12 @@
 From: openSUSE Meeting Reminder <no-reply@opensuse.org>
 To: board@opensuse.org
-Subject: [reminder] Board Conf Call in two days at %(hour)s:%(minute)s NUE Time
+
+Subject: [reminder] Board Conf Call next Tuesday at %(hour)s:%(minute)s NUE Time
```
Comment (cboltz) [inline data/board-meeting.mail:4]: The additional newline will move the subject line into the mail _body_, and the subject will be displayed as "(no subject)". Is this intentional?

Comment (coogor) [reply data/board-meeting.mail:4]: of course not....

**Comment by coogor** [comment]
Versuchen wir es nochmal....

**Pull request merged**